layers/meta-resin-qemu/recipes-bsp/grub/files/grub.cfg

```diff
@@ -2,5 +2,5 @@ default=resin
 timeout=3
 
 menuentry 'resin'{
-linux /bzImage root=/dev/hda2 rootwait
+linux /bzImage root=/dev/disk/by-label/resin-rootA rootwait
```

Comment:
hi @lifeeth you could do it like this instead here: https://github.com/resin-os/resin-intel/blob/master/layers/meta-resin-intel/recipes-bsp/grub/files/grub.cfg_internal-dev#L7